src/encoded/genomic_data_service.py

```diff
@@ -12,34 +22,93 @@
     'tpm': {'title': 'Counts (TPM)'},
     'fpkm': {'title': 'Counts (FPKM)'},
     'assayType': {'title': 'Assay (RNA SubType)'},
+    'biosample_term_name': {'title': 'Biosample term name'},
     'libraryPrepProtocol': {'title': 'Experiment'},
     'expressionID': {'title': 'File'},
-    'annotation': {'title': 'Annotation'},
-    'biosample_term_name': {'title': 'Biosample'},
-    'biosample_classification': {'title': 'Classification'},
+    'annotation': {'title': 'Assembly'},
+    'biosample_classification': {'title': 'Biosample classification'},
     'biosample_sex': {'title': 'Sex'},
     'biosample_organ': {'title': 'Organ'},
     'biosample_system': {'title': 'System'},
     'biosample_summary': {'title': 'Summary'}
 }
 
+
+FACET_TERM_MAP = {
+    'GRCh38': 'Homo sapiens',
+    'mm10': 'Mus musculus',
+}
+
+
+REVERSE_FACET_TERM_MAP = {
```

Comment:
for a small number of values this is ok, but it can be generated automatically:

`{v: k for k, v in FACET_TERM_MAP.items()}`

Reply:
Ha, yeah. This occurred to me after I PRed yesterday.